classes/category/Collector.php

```diff
@@ -159,8 +173,6 @@ public function getQueryBuilder(): Builder
             }
         });
 
-        $qb->orderBy(DB::raw('(COALESCE((pc.seq * 8192) + pc.category_id, 0) * 8192) + CASE WHEN pc.category_id IS NULL THEN 8192 * ((c.seq * 8192) + c.category_id) ELSE c.seq END'));
```

Comment:
The category ordering feature was removed, so I also dropped the `seq` column. As a result, I’ve removed this orderBy clause as well.